esx_service/utils/auth.py

```diff
@@ -34,6 +34,7 @@
 
 SIZE = 'size'
 DEFAULT_TENANT = '_DEFAULT'
+DEFAULT_TENANT_UUID = '_DEFAULT_TENANT_UUID'
```

Comment:
So, this would create a folder "_DEFAULT_TENANT_UUID" and soft link _DEFAULT? Should we be using some constant magic number?